docker-compose.yml

```diff
@@ -1,15 +1,20 @@
 services:
   postgres:
-    image: postgres:16
+    container_name: hostedgpt_postgres
+    build:
+      context: .
+      dockerfile: docker/Dockerfile.postgres
     restart: always
     environment:
-      POSTGRES_USER: app
-      POSTGRES_DB: app_development
-      POSTGRES_PASSWORD: secret
+      - POSTGRES_PASSWORD=${POSTGRES_PASSWORD:-secret}
+      # NOTE: If DATABASE_URL is set it will override values set in HOSTEDGPT_DB_USERNAME and HOSTEDGPT_DB_PASSWORD
+      - DATABASE_URL
+      - HOSTEDGPT_DB_USERNAME=${HOSTEDGPT_DB_USERNAME:-hostedgpt_user}
+      - HOSTEDGPT_DB_PASSWORD=${HOSTEDGPT_DB_PASSWORD:-secret}
```

Comment:
these are now the values for the rails user instead of the default postgres user